devops/dependencies-igc-dev.json

```diff
@@ -1,10 +1,10 @@
 {
   "linux": {
     "igc_dev": {
-      "github_tag": "igc-dev-6ee988a",
-      "version": "6ee988a",
-      "updated_at": "2024-11-26T15:44:10Z",
-      "url": "https://api.github.com/repos/intel/intel-graphics-compiler/actions/artifacts/2239640503/zip",
+      "github_tag": "igc-dev-3db59df",
+      "version": "3db59df ",
```

Comment:
There is one space at the end of value. Will it cause problems?

Reply:
wow nice catch, will fix, thanks!